spec/controllers/review_spec.rb

```diff
@@ -0,0 +1,20 @@
+require_relative '../spec_helper'
+
+describe ReviewsController do
+  describe 'POST create;' do
+    it 'saves a review' do
+      initial_reviews_count = Review.all.count
+      video = Fabricate(:video)
+      review = Fabricate(:review)
+      post :create, {
+                      video_id: video.id,
+                      review: {
+                                body:review.body,
+                                title: review.title,
+                                rating: review.rating
+                              }
+                    }
+      expect(Review.all.count).to be > initial_reviews_count
```

Comment:
You don't need to call `.all` on Review to be able to get the count. `Review.count` is enough.